_posts/en/newsletters/2019-05-07-newsletter.md

```diff
@@ -0,0 +1,273 @@
+      is the [Hardware Wallet Interaction][HWI] (HWI) tool that allows
+      users comfortable working on the command line to use Bitcoin Core
+      with several popular models of hardware wallets.  Internally, the
+      tool makes heavy use of PSBTs and output script descriptors,
+      allowing it to be integrated with other wallets that support those
+      interfaces (e.g. [Wasabi wallet's experimental support][wasabi
+      hwi]).  Work has already begun on more directly integrating HWI
+      with the main Bitcoin Core tools and building graphical interfaces
+      for it.
+
+    - *New wallet tool:* alongside `bitcoind` and other Bitcoin programs
+      is a new `bitcoin-wallet` tool.  This command-line tool currently
+      only allows the user to create a new wallet or perform some basic
+      inspections on an existing wallet, but it's planned to add more
+      features to the tool in subsequent releases.
+
+    - *New architecture and new Ubuntu Snap package:* this is the first
+      release to provide [pre-built binaries][bitcoincore.org download]
+      for Linux on the RISC-V CPU architecture.  For users of Ubuntu and
+      compatible systems, this release also provides a [Snap package][]
+      that replaces the PPA that was updated in previous releases.
+      Both the RISC-V and Snap packages include binaries that are
+      deterministically built and [signed][gitian sigs] by multiple
+      Bitcoin Core contributors.
+
+      {% comment %}<!--
+      152 Tests and QA
+      74 Docs
+      65 wallet
+      55 RPCs and other APIs
+      51 GUI
+      47 Build system
+      43 Misc
+      17 p2p and network code
+      13 Platform support
+      9 block and tx handling
+      1 mining
+      1 consensus
+      -->{% endcomment %}
+
+    - *Numerous testing and Quality Assurance (QA) changes:* the
+      [release notes][0.18 notes] list all the PRs related to 0.18 that
+      Bitcoin Core's release manager thought were significant, split
+      into twelve categories.  Although the significance and
+      categorization criteria are somewhat arbitrary, and the number of
+      PRs doesn't necessarily correlate to the amount of work done, we
+      think it's notable that the "Tests and QA" section in the release
+      notes has more than double the number of PRs listed in any other
+      category.  We don't often get to write about testing in this
+      newsletter---tests are rarely news unless something goes
+      wrong---so we wanted to take this opportunity to remind readers
+      that testing remains an active and important part of Bitcoin Core
+      development.
+
+    - *Plan to switch to bech32 receiving addresses by default:* as
+      mentioned in the *news* section of [Newsletter #40][], the release
+      notes announce the project's intention to switch to bech32 sending
+      addresses by default in either the next major version (0.19,
+      [expected around January 2020][0.19 release schedule]) or the
```

Comment:
This has been updated to November 2019. https://github.com/bitcoin/bitcoin/issues/15940#issuecomment-489921644